js/components/Buttons/PrimaryButton.js

```diff
@@ -14,7 +14,8 @@ const PrimaryButton = ({ title, onPress, disabled }) => (
     buttonStyle={{
       borderRadius: 3,
       backgroundColor: colors.primary,
-      padding: 16,
+      paddingVertical: 16,
```

Comment:
nit:
```suggestion
      padding: 16,
```

incase you have the really long (or text size up) text, you still want the padding horizontal